zbus/src/error.rs

```diff
@@ -20,6 +21,8 @@ pub enum Error {
     Address(String),
     /// An I/O error.
     InputOutput(Arc<io::Error>),
+    /// An I/O error with the related URI.
```

Comment:
Let's be a bit more descriptive here and document the arguments (especially since we don't name them in here, which I know is not ideal).